### ISSUE TITLE
[Bug]: Ender Dust mismatch in recipes, unable to craft some items

### ISSUE THREAD

**Issue by marburps** [FTBTeam/FTB-Modpack-Issues#378]
### Modpack

FTB Endeavour

### Modpack version

1.5.0

### Describe the bug

mismatch between "Ender Dust" and "Ender Pearl Dust". Impossible to create ME wireless booster cards (expect "Ender Dust"), pulverizing "Ender Pearl" as JEI suggests, creates "Ender Pearl Dust".

### Steps to reproduce

Pulverize ender Pearl
Try to create ME wireless booster card with the dust.

### Expected behaviour

Align on a single name for "Ender Dust" / "Ender Pearl Dust" and be consistent through the modpack.

**Comment by marburps** [comment]
Some additional info: looks like using a crusher does give "Ender dust", where the pulverizer gives "Ender pearl dust". So it's not a blocking issue for getting wifi boosters, one can use a Crusher. but then the pulverizer option should disappear from the JEI as proposed methods of obtaining "Ender dust,", since it gives "Ender pearl dust".

**Comment by Raidobw2** [comment]
Thanks, I'll be checking that and fixing it

**Comment by Raidobw2** [comment]
This appears to have been fixed already. The ender pearl dust is no longer used anywhere. "Ender Dust" is used instead. I've added another recipe for the booster with another kind of ender dust, since that one is from better end and is crafted differently, that seems to be the only thing missing.
![image](https://user-images.githubusercontent.com/17010632/148384461-e7b108ec-b672-4f4f-8ec7-333f9545cfcc.png)
